vshavel/forms.html

```diff
@@ -0,0 +1,209 @@
+﻿<!DOCTYPE html>
+
+<html>
+
+	<head>
+
+		<META charset="utf-8"> 
+		<title>Формы</title>
+
+	</head>
+
+	<body>
+
+		<table width='700"'>
+<col="50%"><col="50%">
+			<tr>
+
+				<td>
+
+<h3>Простая форма поиска с вариантами:</h3>
+<form method="get" action="http://sc.it-academy.by/fe/form.php">
+ 	<fieldset>
+ 		<legend>Строка поиска</legend>
+ 		<input type="text" name="search" maxlength="10">
+ 		<input type="submit" value="Отправить"><br>
+ 		<label>
+ 			<input type="radio" name="search_dist" value="site" checked>
+ 			на сайте
+ 		</label>
+ 		<label>
+ 			<input type="radio" name="search_dist" value="google">
+ 			в Google
+ 		</label><br>
+ 		
+ 	</fieldset>
+</form>
+
+   </td>
+   <td>
+
+<h3>Простая форма поиска с вариантами 2:</h3>
+<form method="get" action="http://sc.it-academy.by/fe/form.php">
+ 	<fieldset>
+ 		<legend>Строка поиска</legend>
+
+
+ 		<input type="text" name="search" maxlength="10" required autofocus><span>*</span>
+ 		<input type="hidden" name="hidden_field" value="hidden">
+ 		<input type="submit" value="Отправить"><br>
+		<label>
+ 			<input type="radio" name="search_dist" value="site">
+ 			на сайте
+ 		</label>
+
+ 		<label>  
+ 			<input type="radio" name="search_dist" value="google" checked>
+ 			в Google   
+ 		</label><br> 
+
+ 	</fieldset>  
+
+</form>    
+
+	</td>
+	</tr>
+
+	<tr>
+		<td>
+			<h3>Простая форма авторизации</h3>
+			<form method="post" action="http://sc.it-academy.by/fe/form.php">
+				<fieldset>
+					<legend>Авторизируйтесь</legend>
+					
+					
+		<input type="text" id="name1" name="name"><span>*</span>		
+			<label for="name1">Логин</label><br>
+		
+		<input type="password" id="pass1" name="password"><span>*</span>
+			<label for="pass1">Пароль</label>
+			<label><br>
+				<input type="checkbox" name "check1" id="check1">Оставаться в системе
+			</label><br>
+			<input type="button" name="enter" value="Войти">
+				</fieldset>
+			</form>
+		</td>
+		<td>
+			<h3>Опрос(голосование)</h3>
+			<form method="post" action="http://sc.it-academy.by/fe/form.php">
+			<label>
+				<fieldset>
+					<legend>Как вам?</legend>
+					
+					<input type="radio" name="radio_group"
+					value="radio1" id="radgr1" checked> Плохо
+			</label><br>
+			<label>
+					<input type="radio" name="radio_group
```

Comment:
value of attribute wasn't closed (the quote is missing)

Reply:
can you tell me what do you mean?